fimsauthz-api/fims-authz.yaml

```diff
@@ -66,19 +67,18 @@ paths:
                   type: string
                 scope:
                   description: |
-                    White space separated list of the requested scope(s).
-                    The scope request list size is usually one.  In cases where the list size is not one, the list is verified.  Version 1.0.2 of this specificaiton verifies this set: [dss.read.identification_service_areas, rid.read.enhanced_details]
-                  type: string
+                    White space separated set of the requested scope(s).
+                    The scope request set size is usually one, with an exception.
```

Comment:
Should be more general than "an exception" as there may be others in the future.  Can we reference some permissions/scopes document?

Something more like "set size is usually one.  Requests for multiple scopes must conform to documentation at blah blah."